Userland/Libraries/LibJS/Runtime/IteratorOperations.cpp

```diff
@@ -92,6 +92,20 @@ void iterator_close([[maybe_unused]] Object& iterator)
     TODO();
 }
 
+Vector<Value> iterable_to_list(GlobalObject& global_object, Value iterable, Value method)
+{
+    Vector<Value> values;
```

Comment:
This needs to be a `MarkedValueList` (which is a `Vector<Value>` with some extras) to protect the values from being GC'd during subsequent iterations:

```suggestion
MarkedValueList iterable_to_list(GlobalObject& global_object, Value iterable, Value method)
{
    auto& vm = global_object.vm();
    MarkedValueList values(vm.heap());
```